requirements.txt

```diff
@@ -6,7 +6,7 @@
 # requirements-build.txt
 
 # In-house dependencies
-leaf-common>=1.2.26
+leaf-common>=1.2.27
 leaf-server-common>=0.1.21
```

Comment:
Switch to latest leaf-common with AsyncioExecutor support for initialization step.